README.md

```diff
@@ -829,6 +829,42 @@ export class ReactiveFormComponent implements OnInit {
 }
 ```
 
+### React
+
+React controlled components naturally keep some internal mutable state and update it based on user input via `setState()`.
+
+```js
+import React from 'react'
+
+export default class ReactForm extends React.Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      formValue: ''
```

Comment:
Make it as some more real-life example - based on Angular example with `name` and `email` inputs.